src/wasm-binary.h

```diff
@@ -390,8 +390,8 @@ class WasmBinaryWriter : public WasmVisitor<WasmBinaryWriter, void> {
   void writeMemory() {
     if (wasm->memory.max == 0) return;
     if (debug) std::cerr << "== writeMemory" << std::endl;
-    o << int8_t(BinaryConsts::Memory) << int8_t(log2(wasm->memory.initial))
-                                      << int8_t(log2(wasm->memory.max))
+    o << int8_t(BinaryConsts::Memory) << int8_t(ceil(log2(wasm->memory.initial)))
+                                      << int8_t(ceil(log2(wasm->memory.max)))
```

Comment:
There appears to have been consensus to make these sizes in units of pages. The patch to v8 has 'lgtm' status, and patches have been submitted to sexp-wasm binaryen and the spec. If there is still consensus then it's just a matter of timing.